librz/bin/dwarf.c

```diff
@@ -340,6 +333,44 @@ static const char *dwarf_unit_types[] = {
 	[DW_UT_hi_user] = "DW_UT_hi_user",
 };
 
+RZ_API const char *rz_bin_dwarf_get_tag_name(ut64 tag) {
+	if (tag >= DW_TAG_LAST) {
+		return NULL;
+	}
+	return dwarf_tag_name_encodings[tag];
+}
+
+RZ_API const char *rz_bin_dwarf_get_attr_name(ut64 attr_code) {
+	if (!((attr_code >= DW_AT_sibling && attr_code <= DW_AT_loclists_base) ||
```

Comment:
It just checks whether it falls on an element in `dwarf_attr_encodings`. But actually, other than an upper bound check, this is unnecessary since undefined elements will automatically be NULL anyway. And even more, the array is very sparse so I think I will split it up.